intelsdi-x/snap
# Global tags

Fixes #1280 

Summary of changes:
- Clean up some code
- Apply standard tag before workflow tags
- Support global tags
- Add tests
- Updated documentation

How to verify it:
- each commit can be tested independently (the first 3 commits are just code clean up).
- Run Snap daemon with a config with tags, e.g.:
```json
{
    "control":{
        "tags": {
            "/intel/psutil":{
                "context": "config_example"
            }
        }
    }
}
```
- then run a task and verify published metrics are tagged.
